mod_landing/models.py

```diff
@@ -0,0 +1,21 @@
+"""
+ccextractor-web | models.py
+
+Author   : Saurabh Shrivastava
+Email    : saurabh.shrivastava54+ccextractorweb[at]gmail.com
+Link     : https://github.com/saurabhshri
+
+"""
+
+from mod_dashboard.models import ProcessQueue, ProcessStauts
```

Comment:
Looks like we have a typo in our webapp. Let's solve this in a separate PR later on.

Reply:
Sounds good. 👍